src/wp-includes/media.php

```diff
@@ -1843,6 +1843,19 @@ function wp_filter_content_tags( $content, $context = null ) {
 				$filtered_image = wp_img_tag_add_loading_attr( $filtered_image, $context );
 			}
 
+			/**
+			 * Filters img tag that will be injected into the content.
+			 *
+			 * @since 6.0.0
+			 *
+			 * @param string $filtered_image  the img tag with attributes being created that will
+			 * 									replace the source img tag in the content.
+			 * @param string $context Optional. Additional context to pass to the filters.
+			 *                        Defaults to `current_filter()` when not set.
+			 * @param int    $attachment_id the ID of the image attachment.
+			 */
+			$filtered_image = apply_filters( 'wp_img_tag_add_adjust', $filtered_image, $context, $attachment_id );
```

Comment:
Of the two suggestions `wp_content_img_tag` is clearest to me.

The name needs to avoid confusion between this hook and `get_image_tag`. If a longer name is needed to avoid that confusion then it' not a problem.